### ISSUE TITLE
[Progression]+[Wotlk-default] Incorrect A Distraction for Akama offered

### ISSUE THREAD

**Issue by balleny** [chromiecraft/chromiecraft#6405]
### What client do you play on?

enUS

### Faction

Both

### Content Phase:

70

### Current Behaviour

incorrect A Distraction for Akama offered by default (wotlk).

### Expected Blizzlike Behaviour

there are two different A Distraction for Akama quests.
one should be active while attunements are required (also granting achievement and title) and one that should replace the first one once attunement is no longer required (without title and achievement)

**as the default version has to be changed, the progression version has then be changed, too**

https://wowgaming.altervista.org/aowow/?quest=10985 is currently enabled by default in wotlk client. this quest should not be accessible any longer on default wotlk client. its the progression quest, which should be linked with the achievement and title, but not accessible on default wotlk client. For progression, this should be active.

https://wowgaming.altervista.org/aowow/?quest=13429 is the replacement quest, which should be available instead of the prior one, once no attunement is needed any longer. this quest should not grant achievement or title.

to correct the situation, following steps are necessary:

1. for default wotlk client disable 10985 and enable 13429
2. for progression: disable 13429 and enable 10985 while attunement is required to black temple

### Source

.

### Steps to reproduce the problem

.

### Extra Notes

related
https://github.com/chromiecraft/chromiecraft/issues/6388
https://github.com/chromiecraft/chromiecraft/issues/6404

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/123bf37ee7e6763d9d7f4b778cfe8453577bce6c

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by sinhunter** [comment]
To bring a perspective from live, I have seen several people online with it, but many complaining they completed the quest and have not received the title. 

I have tested on several characters over the past week or so and know for a fact that this to be addressed as soon as possible, before all the characters on the server have completed this quest and only a handful have the title due to using a different wow client if I understand it correctly? 

Seems like the issue is on hold for 7 months now. This quest has been completed on live hundreds of times since Hyjal has been released for over 2 weeks at the time of writing this comment.

I wish I could help with fixing the bug but perhaps someone can change the Quest ID to the TBC one while we are still playing TBC content, which is blizzlike and supposed to be changed later in the WoTLK pre-expansion patch (when DKs are released) on original retail and more recently classic.

Thank you.
